mission-vision-values.md

```diff
@@ -0,0 +1,108 @@
+# OTel Mission, Vision, and Values
+
+## Mission (our overall north star as a community)
```

Comment:
Is the order of vision and mission what we want? I'm used to seeing the mission following a vision since a mission's purpose is to help realize a vision.

Reply:
I guess I'm accustomed (`N=1` here, not saying I'm correct!) that the "Mission" goes first, and is typically very short and easy to memorize/reference. The vision is "below the fold" for those who want to dig deeper.